dynawo/sources/Modeler/Common/DYNSubModel.h

```diff
@@ -62,6 +65,19 @@ class Element;
  * Class interface to each sub model that is used to generate the whole model
  */
 class SubModel {
+ public:
+  /**
+   * @brief Check if a variable is external
+   *
+   * A variable is considered external (Dynawo level) if it is external (dynawo model level) and continuous
+   *
+   * @param var the variable to check
+   * @returns the check status
+   */
+  static inline bool isVariableExternal(const boost::shared_ptr<Variable>& var) {
```

Comment:
Correct

Reply:
After investigation, isExternal == true is only possible in case of continuous variables, by construction in python scripts. This function will be removed